Integrations/MongoDB/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+## [Unreleased]
```

Comment:
no need in changelog for new integration

Reply:
The validator requires it